addons/im_livechat/models/mail_channel_partner.py

```diff
@@ -18,6 +20,6 @@ def _gc_unpin_livechat_sessions(self):
                 SELECT cp.id FROM mail_channel_partner cp
                 INNER JOIN mail_channel c on c.id = cp.channel_id
                 WHERE c.channel_type = 'livechat' AND cp.is_pinned is true AND
-                    cp.write_date < current_timestamp - interval '1 day'
+                    cp.last_seen_date < current_timestamp - interval '1 day'
```

Comment:
Also please take into account that the spec has been updated to only unpin after a day but also only if there are no unread messages for this channel partner.